src/main/java/folk/sisby/tinkerers_smithing/mixin/MixinRegistrySyncManager.java

```diff
@@ -0,0 +1,29 @@
+package folk.sisby.tinkerers_smithing.mixin;
+
+import folk.sisby.tinkerers_smithing.TinkerersSmithing;
+import net.fabricmc.fabric.impl.registry.sync.RegistrySyncManager;
+import net.minecraft.registry.Registry;
+import net.minecraft.util.Identifier;
+import org.jetbrains.annotations.Nullable;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import java.util.Objects;
+
+@Mixin(RegistrySyncManager.class)
+public abstract class MixinRegistrySyncManager {
+	@Redirect(method = "createAndPopulateRegistryMap", at = @At(value = "INVOKE", target = "Lnet/minecraft/registry/Registry;getId(Ljava/lang/Object;)Lnet/minecraft/util/Identifier;"))
+	private static @Nullable <T> Identifier skipRecipes(Registry<T> instance, T t) {
+		final Identifier id = instance.getId(t);
+		if (
```

Comment:
correct me if I'm wrong, can you just use an instanceof ServerRecipe on t here?

Reply:
No, that doesn't seem to work, though I'm not sure why. 
Specifically this is what I tried, in case it was just wrong:
```java
if (t instanceof ServerRecipe<?>) { return null; }
return instance.getId(t);
```

Reply:
Oh, because the serverrecipe serializer is registered, not the recipe itself. I'll try that